stubs/toml/toml/decoder.pyi

```diff
@@ -0,0 +1,78 @@
+import sys
+from _typeshed import StrPath
+from typing import IO, Callable, Generic, MutableMapping, Pattern, Protocol, Text, TypeVar
+
+_T = TypeVar("_T")
+_KT_contra = TypeVar("_KT_contra", contravariant=True)
+_VT = TypeVar("_VT")
+
+if sys.version_info >= (3, 3):
+    FNFError = FileNotFoundError
+else:
+    FNFError = IOError
+
+if sys.version_info >= (3, 6):
+    _PathLike = StrPath
```

Comment:
This doesn't seem right. It's specifically checking for `pathlib.PurePath` or `bytes` or `str`.